From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: 0005 disable li_std_wstring
Date: Mon, 18 May 2020 17:05:27 +0200

disable li_std_wstring to avoid test failure
for BE (ppc64 & s390x) as per bug
https://bugzilla.opensuse.org/show_bug.cgi?id=1171368

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 Examples/test-suite/common.mk |    1 -
 1 file changed, 1 deletion(-)

Index: swig-rel-4.0.1/Examples/test-suite/common.mk
===================================================================
--- swig-rel-4.0.1.orig/Examples/test-suite/common.mk
+++ swig-rel-4.0.1/Examples/test-suite/common.mk
@@ -657,7 +657,6 @@ CPP_STD_TEST_CASES += \
 	li_std_vector_enum \
 	li_std_vector_member_var\
 	li_std_vector_ptr \
-	li_std_wstring \
 	smart_pointer_inherit \
 	template_typedef_fnc \
 	template_type_namespace \
